roles/socle-config/files/socle-config.yaml

```diff
@@ -18,15 +18,26 @@ ingress:
   tls:
     # type supported:
     # - none, tls will be disabled
-    # - acme, cert
-    # - ca, will use a secret you set in cert-manager namespace to build a PKI
+    # - acme, cert-manager handle http01 challenge
+    # - ca, cert-manger will use a secret you set in cert-manager namespace to build a PKI
+    # - tlsSecret, use a wildcard/multi-host tls ke pair for ingresses, cert-manager not involved 
     type: none
     # https://cert-manager.io/docs/configuration/acme/http01/
     acme:
       email: yourpersonal@email.com
     # https://cert-manager.io/docs/configuration/ca/
     ca:
       secretName: a-secret-available # in cert-manager namespace
+    # https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets
+    tlsSecret:
+      # name of the tls secret 
+      name: wildcard-tls
+      # method supported: 
+      # (NOT IMPLEMENTED YET) in-namespace: you are in charge to replicate the secret in tool's namespace
+      # unique: secret is provisionned a namespace and script should replicate it in tool's namespace, 'sourceNamespace' is required
```

Comment:
"secret is provisionned a namespace" -> "secret is provisioned into a namespace"